Enhance and add new features

Likhithsai2580 opened the pull request:
Enhance and add new features to the IntelliJ and VSCode extensions.

**IntelliJ Extension:**
* `ContinuePluginActions.kt`
  - Fix formatting issues.
* `ContinuePluginStartupActivity.kt`
  - Fix formatting issues.
* `ContinuePluginService.kt`
  - Fix formatting issues.
* `ContinueBrowser.kt`
  - Fix formatting issues.
* `ContinuePluginToolWindowFactory.kt`
  - Fix formatting issues.

**VSCode Extension:**
* `commands.ts`
  - Fix potential null reference issue in `fullScreenPanel.reveal()`.
* `VsCodeExtension.ts`
  - Remove unnecessary comments.
  - Fix formatting issues.
* `VsCodeMessenger.ts`
  - Remove unnecessary comments.

ItWasEnder commented on core/index.d.ts:
```diff
@@ -375,6 +375,8 @@ export interface IndexTag extends BranchAndDir {
 }
 
 export interface PearAuth {
+  loginAccessToken?: string;
```
This seems unnecessary as PearAuth is just a type wrapper, also `loginAccessToken` is the same as `accessToken`.

There only exists 1 set of tokens in pearai. That is the session that is created when you first authenticate and that session is revalidated by using the single refresh token.

After each refresh you need to store the new token and refresh code. You can store this in the IDE by using the utils method from the webview.

nang-dev commented:
we need the change to be on landing page

Likhithsai2580 committed:
Enhance and add new features

Enhance and add new features to the IntelliJ and VSCode extensions.

**IntelliJ Extension:**
* `ContinuePluginActions.kt`
  - Fix formatting issues.
* `ContinuePluginStartupActivity.kt`
  - Fix formatting issues.
* `ContinuePluginService.kt`
  - Fix formatting issues.
* `ContinueBrowser.kt`
  - Fix formatting issues.
* `ContinuePluginToolWindowFactory.kt`
  - Fix formatting issues.

**VSCode Extension:**
* `commands.ts`
  - Fix potential null reference issue in `fullScreenPanel.reveal()`.
* `VsCodeExtension.ts`
  - Remove unnecessary comments.
  - Fix formatting issues.
* `VsCodeMessenger.ts`
  - Remove unnecessary comments.

cqdev-co reviewed (changes requested):
I think this comment was added intentionally:

"// Note: This file has been modified significantly from its original contents. New commands have been added, and there has been renaming from Continue to PearAI. pearai-submodule is a fork of Continue (https://github.com/continuedev/continue)."

Let's not remove that

Himanshu-Singh-Chauhan commented:
hey likhithsai, am closing this pr for now as its not updated for the requested changes and has some issues that can't be merged, if you feel the changes are necessary, feel free to reopen it. 
ty for the contribution. :)
